examples/qr-generator-demo.py

```diff
@@ -0,0 +1,20 @@
+import pandas as pd
```

Comment:
Please create `qr-generator-demo` directory and move all these files there.

Reply:
done